packages/cli/src/commands/server/runServer.ts

```diff
@@ -20,22 +20,14 @@ import loadMetroConfig from '../../tools/loadMetroConfig';
 import releaseChecker from '../../tools/releaseChecker';
 
 export type Args = {
-  assetPlugins?: string[];
   cert?: string;
   customLogReporterPath?: string;
   host?: string;
   https?: boolean;
-  maxWorkers?: number;
   key?: string;
-  platforms?: string[];
-  port?: number;
   resetCache?: boolean;
-  sourceExts?: string[];
-  transformer?: string;
   verbose?: boolean;
-  watchFolders?: string[];
   config?: string;
```

Comment:
we should rename it to `metroConfig`, what do you think?

Reply:
Sounds like a plan!